init/systemd/ipa-epn.service.in

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Execute IPA Expired Password Notification (EPN)
```

Comment:
EPN stands for Expiring Password Notification.

Reply:
Fixed, thanks